app/src/main/res/layout/fragment_suggest.xml

```diff
@@ -55,6 +56,7 @@
 
     </android.support.v4.widget.SwipeRefreshLayout>
 
-    <include layout="@layout/fab_tweet_posting"/>
+    <include layout="@layout/fab_tweet_posting"
+        />
```

Comment:
Was this change needed?

Reply:
should I change it again?

Reply:
Can you make it as before as this just removes intending.

Reply:
yeah sure